modules/engine/src/model/model.ts

```diff
@@ -474,13 +485,10 @@ export class Model {
 
   // Update dynamic fields
 
-  /**
-   * Updates the vertex count (used in draw calls)
-   * @note Any attributes with stepMode=vertex need to be at least this big
-   */
-  setVertexCount(vertexCount: number): void {
-    this.vertexCount = vertexCount;
-    this.setNeedsRedraw('vertexCount');
+  /** Specify whether instanced rendering should be used */
+  setInstanced(isInstanced: boolean = true): void {
```

Comment:
Do we need this? Is there a realistic scenario that switches between instanced/non-instance without changing the attribute layout?

Reply:
Good point. In fact, I believe that whether the model is instanced should be detectable from the attribute layout, (`stepMode: 'instance'`). I should to dig into that before landing this.